Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -184,6 +184,15 @@ theorem hasFiniteIntegral_of_bounded [IsFiniteMeasure μ] {f : α → β} {C : 
   (hasFiniteIntegral_const C).mono' hC
 #align measure_theory.has_finite_integral_of_bounded MeasureTheory.hasFiniteIntegral_of_bounded
 
+theorem hasFiniteIntegral_of_fintype [Fintype α] [IsFiniteMeasure μ] {f : α → β} :
+  HasFiniteIntegral f μ :=
+  hasFiniteIntegral_of_bounded (C := (Finset.sup .univ (fun a => ‖f a‖₊) : NNReal)) <| by
+    apply ae_of_all μ
+    intro x
+    rw [← coe_nnnorm (f x)]
+    apply NNReal.toReal_le_toReal
+    apply Finset.le_sup (Finset.mem_univ x)
```

Comment:
```suggestion
    hasFiniteIntegral_of_bounded (C := (Finset.sup .univ (fun a => ‖f a‖₊) : NNReal)) <| by
      apply ae_of_all μ
      intro x
      rw [← coe_nnnorm (f x)]
      apply NNReal.toReal_le_toReal
      apply Finset.le_sup (Finset.mem_univ x)
```

Reply:
General guideline for indentation: proofs are indented by 2 characters, and if a statement requires multiple lines, the first such line must be indented by 4 characters so that it's not flush with the proof. (I'm not sure if this is written anywhere, it looks like the style guide is still written for Lean 3...)